.github/workflows/dummy_lambda.yml

```diff
@@ -1,12 +1,39 @@
 name: deploy-dummy-lambda
 on:
   push:
-    branches:
-      - lambda-deploy-action
+    branches: master
     paths:
       - 'dummy_lambda/**'
+  pull_request:
+    branches: master
 jobs:
+  Test:
```

Comment:
Maybe we should follow a particular naming convention for job names. I see that the job name starts with a small case letter, for example its "build" and not "Build". In this case, maybe you could keep it as "test" instead of "Test"

Reply:
I have fixed this.